frog/imports/api/graphs.js

```diff
@@ -3,8 +3,16 @@ import { uuid } from 'frog-utils'
  
 export const Graphs = new Mongo.Collection('graphs');
 
-export const addActivity = () =>
+export const addGraph = () => {
+  const id = uuid()
```

Comment:
ah, I see you are returning the id... makes sense. carry on :)

if we're doing this in all functions, we could make a little HOF

```
const withID = (fn) => {
    const id = uuid()
    return (
        (args) => {
            fn(...[args] + [id])
            return (id)
        }
    )
}
```

and then we can do 

```
export const addGraph = withId( (id) => 
  Graphs.insert({
    _id: id,
    name: 'untitled',
    created_at: new Date()
    }) 
)
```

just for fun :)